core/src/main/java/org/kohsuke/stapler/Stapler.java

```diff
@@ -802,9 +811,7 @@ boolean tryInvoke(RequestImpl req, ResponseImpl rsp, Object node ) throws IOExce
                     if (!isSocketException(e)) {
                         getServletContext().log("Error while serving " + url, e);
                     }
-                } else if (c.getName().equals("org.acegisecurity.AccessDeniedException")) {
-                    // [HUDSON-4834] A stack trace is too noisy for this; could just need to log in.
-                    // (Could consider doing this for all AcegiSecurityException's.)
+                } else if (EXCEPTIONS_DO_NOT_PRINT_STACKTRACE.contains(c.getName())) {
```

Comment:
Still not enough I think because it does not account for subclasses, like `AccessDeniedException3` in Jenkins core. Would need to walk the inheritance stack. Also I think you could just switch to solely checking the Spring type now, since newer versions of Stapler will only be used with newer versions of Jenkins core.

I am not sure why Stapler is even logging anything here. It is already throwing `ServletException`; should be up to the container or various filters to decide what to do with it.

Reply:
It actually walk through super classes already

Reply:
Oh right, missed that somehow.